packages/generate/src/queries.ts

```diff
@@ -38,6 +45,21 @@ currently supported.`);
 
   console.log(`Detected schema directory: ${params.schemaDir}`);
   const matches = await getMatches(root, params.schemaDir);
+
+  const errs:string[] = [];
```

Comment:
I won't point out each formatting issue, but if you don't mind running `yarn run format`, we use the default prettier rules for formatting.

Reply:
My bad. I was literally just throwing in lines to make things work. Next time (if that will be a thing) I pay attention to that!

Reply:
Still needs a quick pass with the formatter.